galaxy/importer/tests/test_collection_info.py

```diff
@@ -228,90 +228,56 @@ def test_non_null_str_fields(galaxy_col_info):
-        self.metadata['dependencies'].update({'alice.php': '1.2.3'})
+def test_self_dependency(galaxy_col_info):
+    namespace = galaxy_col_info['namespace']
+    name = galaxy_col_info['name']
+    galaxy_col_info['dependencies'] = {
+        '{}.{}'.format(namespace, name): '1.0.0'
+    }
+    with pytest.raises(ValueError) as exc:
+        GalaxyCollectionInfo(**galaxy_col_info)
+    assert 'Cannot have self dependency' in str(exc)
+
+
+def test_dep_bad_version_spec(galaxy_col_info):
+    bad_version_specs = [
+        {'alice.apache': 'bad_version'},
+        {'alice.apache': '1.2.*'},
+        {'alice.apache': ''},
+        {'alice.apache': '*.*.*'},
+        {'alice.apache': '>=1.0.0, <=2.0.0'},
+        {'alice.apache': '>1 <2'},
+    ]
+    for dep in bad_version_specs:
+        galaxy_col_info['dependencies'] = dep
         with pytest.raises(ValueError) as exc:
-            GalaxyCollectionInfo(**self.metadata)
-        assert 'collection not in galaxy' in str(exc)
-
-    def test_dep_bad_version_spec(self):
-        bad_version_specs = [
-            {'alice.apache': 'bad_version'},
-            {'alice.apache': '1.2.*'},
-            {'alice.apache': '>=1.0.0, <=2.0.0'},
-        ]
-        for dep in bad_version_specs:
-            self.metadata['dependencies'] = dep
-            with pytest.raises(ValueError) as exc:
-                GalaxyCollectionInfo(**self.metadata)
-            assert 'version spec range invalid' in str(exc)
-
-    def test_dep_cannot_find_ver(self):
-        missing_versions = [
-            {'alice.apache': '2'},
-            {'alice.apache': '1.0.1'},
-            {'alice.apache': '>1.0.0'},
-            {'alice.apache': '!=1.0.0'},
-            {'alice.apache': '~1'},  # semantic_version error
-            {'gunjan.gunicorn': '<1.2.3'},
-            {'gunjan.gunicorn': '1.5'},
-            {'gunjan.gunicorn': '^1.5'},
-        ]
-        for dep in missing_versions:
-            self.metadata['dependencies'] = dep
-            with pytest.raises(ValueError) as exc:
-                GalaxyCollectionInfo(**self.metadata)
-            assert 'no matching version found' in str(exc)
-
-    def test_dep_success(self):
-        good_deps_ver_ranges = [
-            {'alice.apache': '1.0.0'},
-            {'alice.apache': '*'},
-            {'alice.apache': '^1.0'},
-            {'alice.apache': '>0.9.1'},
-            {'gunjan.gunicorn': '1.2.3'},
-            {'gunjan.gunicorn': '1.2.4'},
-            {'gunjan.gunicorn': '^1.1'},
-            {'gunjan.gunicorn': '!=1.2.3'},
-            {'gunjan.gunicorn': '>=1.0.0,<=2.0.0'},
-            {'gunjan.gunicorn': '>=1.0.0,!=1.0.5'},
-        ]
-        for dep in good_deps_ver_ranges:
-            self.metadata['dependencies'] = dep
-            res = GalaxyCollectionInfo(**self.metadata)
-            assert isinstance(res, GalaxyCollectionInfo)
+            GalaxyCollectionInfo(**galaxy_col_info)
+        assert 'version spec range invalid' in str(exc)
```

Comment:
Why did these not move with the other tests?

Reply:
These test the checks that continue to occur in the importer, which check if the version range spec is valid